### PR TITLE
Feature/tabs

### PR DESCRIPTION
**Pull request by kevinch**
screen shot 2018-05-25 at 15 09 14

### CODE REVIEW

**Comment by kevinch** [comment]
closes https://github.com/vtex/styleguide/issues/58

**Comment by kevinch** [comment]
TODO: update readme & version before merge.

File: src/components/Tabs/index.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+          {React.Children.map(children, (child, index) => {
+            const style =
+              index === activeIndex
+                ? { boxShadow: `0 2px ${config.colors['red']}` }
+                : {}
+
+            return (
+              <button
+                type="button"
+                onClick={() => this.handleClick(index)}
```
Comment (klzns) [inline src/components/Tabs/index.js]: Isso deve chamar uma prop `this.props.onClick`

File: src/components/Tabs/index.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+import PropTypes from 'prop-types'
+
+import config from 'vtex-tachyons/config.json'
+
+class Tabs extends Component {
+  constructor(props) {
+    super()
+
+    this.state = {
+      activeIndex: props.activeIndex,
```
Comment (klzns) [inline src/components/Tabs/index.js]: Styleguide tem que ter so dumb components

Comment (klzns) [reply src/components/Tabs/index.js]: https://medium.com/@dan_abramov/smart-and-dumb-components-7ca2f9a7c7d0

File: src/components/Tabs/index.js (hunk 1 above)
Comment (klzns) [reply src/components/Tabs/index.js]: Isso!

**Pull request merged**